### DISCUSSION TITLE
pihole does ratelimit dot proxy

### DISCUSSION THREAD

**Discussion by LuckyTurtleDev** [pi-hole/pi-hole#4643] [Bugs]
I have only nginx as client, witch is used as dot proxy. So pi-hole thinks every query does come from the same client and does rate limit it. 
How can I disable the rate limit?

![image](https://user-images.githubusercontent.com/44570204/157299595-d4b70dcd-37a1-4ea7-aaf9-85f0011403c6.png)

**Comment by jfb-pihole** [comment]
https://docs.pi-hole.net/ftldns/configfile/#rate_limit

**Reply by LuckyTurtleDev** [reply]
Thanks that has fix the issue.
